src/engine-actions/get-max-channel.js

```diff
@@ -0,0 +1,22 @@
+const { Big } = require('../utils')
+const { listChannels } = require('../lnd-actions')
+
+/**
+ * Get local balance of all channels for a specific daemon
+ *
+ * @return {String} totalBalance (int64)
+ */
+async function getMaxChannel ({ outbound = true } = {}) {
+  const { channels = [] } = await listChannels({ client: this.client })
+  const balance = outbound ? 'localBalance' : 'remoteBalance'
+
+  if (channels.length === 0) {
+    this.logger.debug('getMaxChannel: No channels exist')
+    return {}
+  }
+
+  const maxBalance = channels.reduce((max, channel) => Big(channel[balance]).gte(max) ? Big(channel[balance]) : max, Big(channels[0][balance]))
```

Comment:
[nit] Semantic change: since we are looking for max balance, we should use `gt` instead of `gte`